crates/apollo-compiler/test_data/diagnostics/0091_recursive_interface_definition.graphql

```diff
@@ -4,13 +4,17 @@ interface A implements B {
 interface B implements A {
   name: String
 }
-fragment recursive on A {
-  name
+type Impl implements A & B {
+  name: String
 }
```

Comment:
Unrelated change, just fixing the schema so it doesn't rely on having an interface with 0 implementers